### PR TITLE
Enable https://eslint.org/docs/rules/no-console

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: src/debug.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const debug = {
```
Comment (michielbdejong) [inline src/debug.ts]: This seems to work, right?

**Comment by michielbdejong** [comment]
@megoth as discussed, let's wait for you to discuss this with @timbl before approving.

**Comment by michielbdejong** [comment]
@megoth any progress on this? I would like to get this merged before the end of the week, so it doesn't drag on too much.

File: src/acl/acl.ts [hunk 2]
```diff
@@ -373,7 +374,7 @@ export function fixIndividualCardACL (person: $rdf.NamedNode, log: Function, cal
  * This function is used by [[fixIndividualCardACL]]
  */
 export function fixIndividualACL (item: $rdf.NamedNode, subjects: Array<$rdf.NamedNode>, log: Function, callbackFunction: Function): void {
-  log = log || console.log
+  log = log || debug.log
```
Comment (megoth) [inline src/acl/acl.ts]: Think this could be set as default param for the log parameter, but of course that begs to ask what we should set as default value for callbackFunction. Nevertheless something we want to clean up later, to not make this PR too extensive.

File: src/debug.ts (hunk 1 above)
Comment (megoth) [reply src/debug.ts]: You could also just export each funtion separately, e.g. `export function log (...args: any[])` instead of packing it up in an object.

File: src/debug.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  log (...args: any[]) {
+    // eslint-disable-next-line no-console
+    console.log(...args)
```
Comment (megoth) [inline src/debug.ts]: Had forgotten that you can spread arguments to a function call that way. Nifty.

File: src/widgets/index.js [hunk 4]
```diff
@@ -28,6 +30,6 @@ var widgets = Object.assign(
 )
 
 module.exports = widgets
-console.log('@@@ widgets: ', widgets)
+debug.log('@@@ widgets: ', widgets)
```
Comment (megoth) [inline src/widgets/index.js]: We need to check with Tim how valuable this line actually is currently. It's really annoying in tests.

File: test/unit/helpers/clearStore.ts [hunk 5]
```diff
@@ -1,9 +1,9 @@
 import store from '../../../src/store'
```
Comment (megoth) [inline test/unit/helpers/clearStore.ts]: Lets revert the changes to this file? Think we have a much better solution in master branch.

File: src/widgets/index.js (hunk 4 above)
Comment (michielbdejong) [reply src/widgets/index.js]: Agree! :)

File: src/debug.ts (hunk 1 above)
Comment (michielbdejong) [reply src/debug.ts]: Done in c5f47f0
